src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -53,6 +53,10 @@ public sealed class Arguments : TransformInputBase
         private const string RegistrationName = "TensorFlowTransform";
 
         internal readonly TFSession Session;
```

Comment:
>internal [](start = 8, length = 8)

Do all these fields need to be internal?

Reply:
I refactored this, so we have 2 private fields (_savedModelPath and _isTemporarySavedModel)

---
In reply to: [219205265](https://github.com/dotnet/machinelearning/pull/853#discussion_r219205265) [](ancestors = 219205265)

Reply:
I think the other ones can be private too, it looks like they are only used by TensorFlowTransform and TensorFlowTransform.Mapper, which should be able to access TensorFlowTransform's private fields.

---
In reply to: [219275115](https://github.com/dotnet/machinelearning/pull/853#discussion_r219275115) [](ancestors = 219275115,219205265)